docs/Segments.md

```diff
@@ -118,6 +118,91 @@ This is platform specific; parses the data and interprets as a header for e.g. N
   start: 0xABC
 ```
 
+## `data`
+
+**Description:**
+
+Data located in the ROM.
```

Comment:
The `data` segment will decompile this section of the ROM as data, rather than instructions, and write out assembly files with a `.data.s` suffix.